snippets/base/urls.py

```diff
@@ -7,25 +7,18 @@
 
 
 urlpatterns = [
-    url(r'^$', views.SnippetIndexView.as_view(), name='base.index'),
-    url(r'^(?P<startpage_version>[^/]+)/(?P<name>[^/]+)/(?P<version>[^/]+)/'
```

Comment:
loving how you can cast url variables now. Here making `startpage_version` an int.

Reply:
Oooh!

Reply:
Ooooooooo! So this is something the new `django.urls.path` can do? I need to read up on this. Looks nice! A bit more Flasky.

Reply:
Oh yes. It's very nice. https://docs.djangoproject.com/en/2.1/topics/http/urls/#path-converters

Need to get a project up to Django 2.0+ :)